package.json

```diff
@@ -33,6 +33,7 @@
     "unit": "tap test/*.js",
     "unit-file": "tap",
     "linkall": "node ./scripts/linkall.js",
+    "unlinkall": "find . -name a -type d -exec rm -rf {} \\;",
```

Comment:
Yes, but I think we need to do it differently. I think we need to list all linked mendel-* modules and remove them. For instance, when switching from v2 to master we should be able to unlink the v2 modules without having them on the tree, right?